deploy/kubernetes/deploy-driver.sh

```diff
@@ -16,6 +16,29 @@ set -o errexit
 readonly PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
 readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
 
+function check_service_account()
+{
+	# Using bash magic to parse JSON for IAM
+	IAM_NAME=$(grep -Po '"client_email": *\K"[^"]*"' ${GCE_PD_SA_DIR}/cloud-sa.json | tr -d '"')
+	PROJECT=$(grep -Po '.*@\K[^.]+'<<<$IAM_NAME)
+	GOTTEN_BIND_ROLES=$(gcloud projects get-iam-policy $PROJECT --flatten="bindings[].members" --format='table(bindings.role)' --filter="bindings.members:$IAM_NAME")
```

Comment:
nit: all of these variables are readonly. I don't really know bash style guidelines but I'd like it to be at least internally consistent.